android-components/gradle.properties

```diff
@@ -40,8 +40,8 @@ android.defaults.buildfeatures.shaders=false
 
 # Default values for pom.xml
 libRepositoryName=Mozilla-Mobile
-libUrl=https://github.com/mozilla-mobile/android-components
-libVcsUrl=https://github.com/mozilla-mobile/android-components.git
+libUrl=https://github.com/mozilla-mobile/firefox-android
```

Comment:
Actually, should this be `https://github.com/mozilla-mobile/firefox-android/android-components`?

Reply:
Hm I think this should really go to the website of the library, but I don't want to link to `mozac.org` because we want to migrate everything to firefox-source-docs eventually.

So, yes, let's link to the README which is `https://github.com/mozilla-mobile/firefox-android/android-components` as you suggested. :)

Reply:
Updated to `https://github.com/mozilla-mobile/firefox-android/tree/main/android-components`